ch390/src/esp_eth_phy_ch390.c

```diff
@@ -116,25 +93,16 @@ static esp_err_t ch390_loopback(esp_eth_phy_t *phy, bool enable)
     phy_802_3_t *phy_802_3 = esp_eth_phy_into_phy_802_3(phy);
```

Comment:
I think we could delete `ch390_loopback` and use just `802_3` loopback function https://github.com/espressif/esp-idf/blob/master/components/esp_eth/src/phy/esp_eth_phy_802_3.c#L350